conans/client/generators/b2.py

```diff
@@ -108,8 +108,8 @@ def b2_constants_for_dep(self, name, info, user=None):
             self.b2_constant(name, 'exelinkflags', info.exelinkflags) + \
             self.b2_constant(name, 'requirements', self.b2_features(self.b2_variation)) + \
             self.b2_constant(name, 'usage-requirements', [
-                '<include>$(includedirs({name},{variation}))'.format(name=name, variation=self.b2_variation_id),
```

Comment:
Whas the "-" not correct? Is it related?

Reply:
Oh, this change should not be there, I'll fix as soon as I am near a computer.

Reply:
Done.